SConstruct

```diff
@@ -1003,6 +1003,39 @@ if env["ARNOLD_ROOT"] :
 # Definitions for the libraries we wish to build
 ###############################################################################################
 
+# When Cycles is built, it uses several preprocessor variables that enable and
```

Comment:
Indeed this was real frustration - I do have a patch for them to merge that fills out a proper config header via CMake, but until then this is the best we can do...